src/main/java/pl/creative/rental_server/db/entities/Place.java

```diff
@@ -24,10 +24,13 @@ public class Place {
     private String name;
     private String description;
 
-//    @ToString.Exclude
+    //    @ToString.Exclude
```

Comment:
Better option than excluding in to string method is to use correct property mapping . https://www.projectlombok.org/features/ToString